src/osmf/js/osmf_flash.js

```diff
@@ -391,6 +391,7 @@ require("../../../html5-common/js/utils/constants.js");
      */
     this.getCurrentTime = function() {
       this.callToFlash("getCurrentTime");
+      return currentTime;
```

Comment:
so when the return happens it's already updated?  Does the callback get called synchronously?